### PR TITLE
student view available

### PR DESCRIPTION
**Pull request by alexandriainfantino**
I wasn't entirely sure how getting grades for courses is going to be handled with the API. lmk if this is weird

### CODE REVIEW

**Comment by yura-kim** [comment]
I just created a pull request with a bunch of changes to the API facade since we're doing sections now. Could you use those changes as a base for these?

**Comment by alexandriainfantino** [comment]
I think I can use the getGradeAndCommentsForSection function once that gets merged in

**Comment by alexandriainfantino** [comment]
don't merge yet, fiddling with some stuff

**Review by ShaydeNofziger — APPROVED** [review]

**Pull request merged**
